src/recommendations/RecommendationCard.jsx

```diff
@@ -39,6 +39,7 @@ const RecommendationCard = (props) => {
       <Hyperlink
         destination={recommendation.marketingUrl}
         target="_blank"
+        id="card-box"
```

Comment:
Don't use ID over here because this component is iterating from the `RecommendationCardList` component and this may raise a warning of the same ID for multiple elements.

Reply:
addressed